app/javascript/components/common/SearchableList.tsx

```diff
@@ -131,7 +131,7 @@ const Results = ({
           setOrder={setOrder}
         />
       ) : null}
-      {latestData ? (
+      {latestData && latestData.meta.totalPages > 0 ? (
```

Comment:
Should this not be `> 1`? If there's only one result, totalPages is still `1` I think?

Also, can we add a test for this pls.

Reply:
I think you're right :)

Reply:
Oh yes!